source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ColumnRenderer.java

```diff
@@ -67,9 +69,36 @@ else if ((column = Column.dynamic_cast(baseCardElement)) == null)
         ContainerStyle styleForThis = column.GetStyle().swigValue() == ContainerStyle.None.swigValue() ? containerStyle : column.GetStyle();
         LinearLayout returnedView = new LinearLayout(context);
         returnedView.setOrientation(LinearLayout.VERTICAL);
+
+        LinearLayout verticalContentAlignmentLayout = new LinearLayout(context);
+        verticalContentAlignmentLayout.setOrientation(LinearLayout.HORIZONTAL);
+        verticalContentAlignmentLayout.setLayoutParams(new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.MATCH_PARENT));
+
+        VerticalContentAlignment contentAlignment = column.GetVerticalContentAlignment();
+        if(contentAlignment != VerticalContentAlignment.Stretch)
+        {
+            if(contentAlignment == VerticalContentAlignment.Top)
+            {
+                verticalContentAlignmentLayout.setGravity(Gravity.TOP);
+            }
+            else if( column.GetVerticalContentAlignment() == VerticalContentAlignment.Center )
+            {
+                verticalContentAlignmentLayout.setGravity(Gravity.CENTER_VERTICAL);
+            }
+            else if( column.GetVerticalContentAlignment() == VerticalContentAlignment.Bottom )
+            {
+                verticalContentAlignmentLayout.setGravity(Gravity.BOTTOM);
+            }
+        }
```

Comment:
Should this be a switch statement? At least make it so that you don't query the property in the other cases and use the contentAlignment variable.